rpcs3/Emu/system_config.h

```diff
@@ -52,6 +52,7 @@ struct cfg_root : cfg::node
 		cfg::_enum<tsx_usage> enable_TSX{ this, "Enable TSX", enable_tsx_by_default() ? tsx_usage::enabled : tsx_usage::disabled }; // Enable TSX. Forcing this on Haswell/Broadwell CPUs should be used carefully
 		cfg::_bool spu_accurate_xfloat{ this, "Accurate xfloat", false };
 		cfg::_bool spu_approx_xfloat{ this, "Approximate xfloat", true };
+		cfg::_bool spu_relaxed_xfloat{ this, "Relaxed xfloat", true }; // Approximate accuracy for only the "FCGT" and "FNMS" instructions
```

Comment:
we now have 3 xfloat settings.
wouldn't it make more sense to use an enum?

Reply:
Neko wanted accurate and approximate to be separate settings, but maybe I can merge relaxed and approx.

Reply:
I think it's fine,